### PR TITLE
Concerned About Separation excercise + update format for new workshop…

### PR DESCRIPTION
**Pull request by AydinHassan**
TODO:
- [x] Tests
- [x] Update workshop dependency
- [x] Fix DirectoryFilter

### CODE REVIEW

File: exercises/concerned-about-separation/problem/problem.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+This problem is the same as the previous but introduces the concept of **classes**. You will need to create two files to solve this.
```
Comment (mikeymike) [inline exercises/concerned-about-separation/problem/problem.md:1]: Maybe add a reference to the specific task instead and tell them to complete that first if they haven't already? 

I'm thinking this as there is no specific order tasks have to be done in.

Comment (mikeymike) [inline exercises/concerned-about-separation/problem/problem.md:1]: O wait you've explained it again anyway so probably not needed really.

File: test/res/concerned-about-separation/include.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+require_once __DIR__ . '/some-other-file.php';
+
+echo "No Include yo!";
```
Comment (mikeymike) [inline test/res/concerned-about-separation/include.php]: you sure :laughing:

Comment (AydinHassan) [inline test/res/concerned-about-separation/include.php]: damn

**Pull request merged**